module-system/sov-modules-stf-template/clippy.toml

```diff
@@ -0,0 +1 @@
+type-complexity-threshold  = 300
```

Comment:
`clippy` was not happy with this signature (return type):
```

    fn execute_txs(
        &mut self,
        mut batch_workspace: WorkingSet<C::Storage>,
        txs: Vec<TransactionAndRawHash<C>>,
        batch_data_and_hash: &BatchDataAndHash,
    ) -> Result<(WorkingSet<C::Storage>, Vec<TransactionReceipt<TxEffect>>)> {
...
}
```
But I don't thing it is that bad so I increased the `type-complexity-threshold`